programs/account-compression/src/instructions/rollover_address_merkle_tree_and_queue.rs

```diff
@@ -49,37 +50,36 @@ impl<'info> GroupAccounts<'info> for RolloverAddressMerkleTreeAndQueue<'info> {
 pub fn process_rollover_address_merkle_tree_and_queue<'a, 'b, 'c: 'info, 'info>(
     ctx: Context<'a, 'b, 'c, 'info, RolloverAddressMerkleTreeAndQueue<'info>>,
 ) -> Result<()> {
-    assert_size_equal(
-        &ctx.accounts.old_queue.to_account_info(),
-        &ctx.accounts.new_queue.to_account_info(),
-    )?;
-    assert_size_equal(
-        &ctx.accounts.old_address_merkle_tree.to_account_info(),
-        &ctx.accounts.new_address_merkle_tree.to_account_info(),
-    )?;
```

Comment:
Please revert this deletion. This check asserts that the forester who performs the rollover doesn't change the account size.